.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Type: bug'
+assignees: mastercoms
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**How you encountered the issue**
+Tell us the steps on how to recreate the bug.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+* Team Comtress version (don't say latest, say the actual version):
+* .NET Core version:
+* DepotDownloader version:
+* Operating system (OS):
+* Graphics card (GPU):
```

Comment:
We also don't really care about people's specs. Might be better to include a dxlevel, mods used if any, etc. OS is also always Windows, so maybe default to that.

Reply:
i readded launch options and dx level, also added mods. Ss for specs, things like gpu can definitely afffect this. Windows what? there are still people using 7,8,8.1 and different versions of 10

Reply:
Thanks for adding dxlevel and mods, but I still think specs aren't needed here

Reply:
I think we need opinion from real pro @mastercoms to help us settle this argument

Reply:
Yeah I think specs are very helpful

Reply:
Great, another conversation resolved.